lists.go

```diff
@@ -25,3 +30,19 @@ func (s *Session) GetUsers(l *List) (users []User, err error) {
 	err = s.do(rq).Into(&users)
 	return
 }
+
+// OptIn subscribes the given user to the given list
+// This consumes one API request.
+func (s *Session) OptIn(UserID int, ListID int) (err error) {
+	var body bytes.Buffer
+	var ok *bool
+	body.WriteString(fmt.Sprintf("userid=%d", UserID))
```

Comment:
If you're creating a `bytes.Buffer` and immediately adding a `string` into it, maybe use [`bytes.NewBufferString`](https://golang.org/pkg/bytes/#NewBufferString), something like:

```go
body := bytes.NewBufferString(fmt.Sprintf("userid=%d", UserID))
```